src/korean.js

```diff
@@ -93,6 +96,12 @@ class Korean {
     } // end of first else
   } // end of presentWord function
 
+  doPresentContinuous(word) {
+    let wordLength = word.length;
+    let conjugate = word.substring(0, wordLength-1);
+    return `${conjugate}고있어`;
```

Comment:
Wow, I'm amazed how my initial 5 - 7 liner turned into a one liner. Please let me know if there are any more issues that I could help out with. Thanks!